charts/team-ns/templates/builds/buildpack.yaml

```diff
@@ -29,12 +29,25 @@ spec:
       value: {{ .mode.buildpacks.repoUrl }}
     - name: revision
       value: {{ .mode.buildpacks.revision }}
+  - name: grype-code-scan
+    taskRef:
+      name: grype
+      kind: Task
+    runAfter:
+      - fetch-source
+    params:
+      - name: ARGS
+        value:
+          - dir:.
```

Comment:
users should be able to specify where their source code is.

For example for Docker build we have specified `$context` because some repos may contain code for different apps

Reply:
The scan job comes after the `git-clone` task and scans the root of the repoURL used in the clone task.